examples/nextjs/pages/pages/[handle].js

```diff
@@ -1,11 +1,17 @@
-import React from 'react';
+import React, { useContext } from 'react';
```

Comment:
`useEffect` also needs to be imported:

<img width="1055" alt="Screen Shot 2021-03-02 at 1 05 14 PM" src="https://user-images.githubusercontent.com/5732000/109693915-331fe980-7b58-11eb-9267-7590c6648df7.png">

Reply:
Good catch! Strange it was working when I tested it with logs...

Should be good now thank you!